components/permit-holders/AppHistoryCard.tsx

```diff
@@ -0,0 +1,111 @@
+import { Box, Link, Text, Wrap, Divider } from '@chakra-ui/react'; // Chakra UI
+import Table from '@components/internal/Table'; // Table component
+import RequestStatusBadge from '@components/internal/RequestStatusBadge'; // Request status badge component
+import PermitHolderInfoCard from '@components/internal/PermitHolderInfoCard'; // Custom Card Component
+
+// Placeholder data
+
+const DATA = Array(4).fill({
+  rcdPermitId: { rcdPermitId: 354 },
+  isRenewal: 'false',
+  requestStatus: 'EXPIRING',
+  expiryDate: '2021/01/01',
+  applicationId: { applicationId: 1 },
+});
+
+const COLUMNS = [
+  {
+    Header: 'Permit #',
+    accessor: 'rcdPermitId',
+    disableSortBy: true,
+    maxWidth: 140,
+    Cell: _renderPermitId,
+  },
+  {
+    Header: 'Request Type',
+    accessor: 'isRenewal',
+    disableSortBy: true,
+    maxWidth: 140,
+    Cell: _renderRequestType,
+  },
+  {
+    Header: 'Status',
+    accessor: 'requestStatus',
+    disableSortBy: true,
+    maxWidth: 190,
+    Cell: _renderStatusBadge,
+  },
+  {
+    Header: 'Expiry Date',
+    accessor: 'expiryDate',
+    disableSortBy: true,
+    maxWidth: 140,
+  },
+  {
+    accessor: 'applicationId',
+    disableSortBy: true,
+    maxWidth: 140,
+    Cell: _renderAppLink,
+  },
+];
+
+type PermitIdProps = {
+  value: { rcdPermitId: number };
+};
+
+function _renderPermitId({ value }: PermitIdProps) {
+  return (
+    <>
+      <Text as="p">{`#${value.rcdPermitId}`}</Text>
+    </>
+  );
+}
+
+type RequestTypeProps = {
+  value: { isRenewal: boolean };
+};
+
+function _renderRequestType({ value }: RequestTypeProps) {
+  return (
+    <>
+      <Text as="p">{value.isRenewal ? `Renewal` : `Replacement`}</Text>
+    </>
+  );
+}
+
+type StatusProps = {
+  value: 'COMPLETED' | 'INPROGRESS' | 'PENDING' | 'REJECTED' | 'EXPIRING' | 'EXPIRED' | 'ACTIVE';
```

Comment:
There's now an `ApplicationStatus` enum which was added by Oustan recently so it would be great if we could use that here instead. I know it needs to still be fixed in other places too so if its not trivial to only use it here then maybe don't worry about it and it can be fixed separately.

Reply:
The enum Oustan added has different values than the values here (enum values: PENDING, APPROVED, REJECTED, COMPLETED). I think APPROVED == INPROGRESS and the rest are calculated based on the expiry date. 🤔 I'm not entirely sure how to handle that atm, i think i might just address it separately later

Reply:
cc @OustanDing